src/extension.ts

```diff
@@ -42,7 +42,7 @@ export function activate(context: vscode.ExtensionContext) {
 }
 
 function isAsyncAPIFile(text: string) {
-  return text.includes('asyncapi:');
+  return text.includes('asyncapi:') || text.includes('"asyncapi"');
```

Comment:
you have no `:` in `'"asyncapi"'`

maybe instead we should have `text.startsWith`? I know that AsyncAPI document does not have a requirement to start with `asyncapi`, but I'm just afraid `text.includes('"asyncapi"')` is to generic and can appear in many different files

Reply:
yes, I updated the code to do a proper match.. (parsing the json and matching the start of line for yml)
This only affects the menu visibility for "AsyncAPI Preview" but it's better a proper match..